sigs/recommenders/CHARTER.md

```diff
@@ -22,7 +22,7 @@ Notice that TensorFlow has open-sourced [TensorFlow Recommenders](https://blog.t
 [github.com/tensorflow/recommenders](http://github.com/tensorflow/recommenders)
 
 Further, we maintain a Tensorflow repo dedicated for community contributions and maintained by SIG as well, under: 
-**github.com/tensorflow/recommenders-addons** (to be created).
+**github.com/tensorflow/recommenders-addons**.
```

Comment:
Would it make sense to turn this into a proper URL, now that the repository exists?

I.e., https://github.com/tensorflow/recommenders-addons